README.md

```diff
@@ -64,6 +64,17 @@ interface CreateTaskOptions {
   interval?: number;        // execution interval in seconds
   startTime?: number;       // start timestamp in seconds or 0 to start immediately (default: 0)
 
+  // Proxy caller
+  proxy?: boolean;          // executions will be routed through proxy for extra safety.
+
+  // Single execution task
+  singleExec?: boolean;     // task cancels itself after 1 execution if true.
+
+  // Polywrap params
+  polywrapHash?: string;    // ipfs hash of polywrap.
+  polywrapArgs?: 
```

Comment:
We should maybe name those parameter more like "off-chain resolver" instead of `polywrap`

```suggestion
  // Off-chain resolver params (using polywrap)
  offChainResolverHash?: string;    // ipfs hash of your polywrap resolver.
  offChainResolverArgs?: 
```